Introduce `lua_eval`, `lua_call`, and `sql` grant object types (#8906)
tarantool/tarantool: changelogs/unreleased/gh-8803-exec-priv.md

## feature/box

* Introduced the new `lua_eval` and `lua_call` object types for

>   `box.schema.user.grant`. Granting the 'execute' privilege on `lua_eval`
>   permits the user to execute an arbitrary Lua expression with the
>   `IPROTO_EVAL` request. Granting the 'execute' privilege on `lua_call`
>   permits the user to execute any global user-defined Lua function with
>   the `IPROTO_CALL` request (gh-8803).
> * **[Breaking change]** Introduced the new `sql` object type for
>   `box.schema.user.grant`. Now, only users that have the 'execute' privilege
>   granted on `sql` or `universe` may execute SQL expressions with the

Reviewer: ```suggestion
  `box.schema.user.grant`. Granting the `'execute'` privilege on `lua_eval`
  allows the user to execute an arbitrary Lua expression with the
  `IPROTO_EVAL` request. Granting the `'execute'` privilege on `lua_call`
  allows the user to execute any global user-defined Lua function with
  the `IPROTO_CALL` request (gh-8803).
* **[Breaking change]** Introduced the new `sql` object type for
  `box.schema.user.grant`. Now only users with the 'execute' privilege
  granted on `sql` or `universe` can execute SQL expressions with the
```

Author: Fixed. Thanks.
